src/rpc.cpp

```diff
@@ -133,6 +134,27 @@ RPCServer::RPCServer() {
         RPCObjectID realm_id = store_object(realm_object);
         return (json){{"result", realm_id}};
     };
+    m_requests["/create_user"] = [this](const json dict) {
```

Comment:
Looks like you are already calling `store_object` in `serialize_json_value`, so the object should be stored.